app/export_config/generate_form.py

```diff
@@ -269,6 +271,8 @@ def build_form_json(form: Form) -> dict:
     # Build the basic page structure
     for page in form.pages:
         results["pages"].append(build_page(page=page))
+        if page.section:
+            results["sections"].append(page.section)
```

Comment:
Is this right? The `section` property on the page is just a string, but the `sections` property on a form is a list of objects, which have a name, title etc

```
    "sections": [
        {
            "name": "aucbfI",
            "title": "Section 1",
            "hideTitle": false
        },
        {
            "name": "vJLJeJ",
            "title": "Section 2",
            "hideTitle": false
        }
    ]